core/src/test/java/org/arquillian/smart/testing/configuration/ConfigurationUsingPropertyTest.java

```diff
@@ -176,7 +176,7 @@ public void should_load_configuration_with_defaults_and_with_specified_system_pr
         expectedConfiguration.setScm(scm);
 
         // when
-        final Configuration actualConfiguration = ConfigurationLoader.load();
+        final Configuration actualConfiguration = ConfigurationLoader.load(Paths.get("").toFile());
```

Comment:
in these test classes, when it is used several times, wouldn't it be better to call `Paths.get("").toFile()` only once and assign it to some global variable such as `currentDir`...?